### PR TITLE
Update new-repo-checklist.md

### PR DESCRIPTION
**Pull request by lseelenbinder**
* Add note about what tier the project will assume when onboarded
* Clarifies language around what board member approval looks like

### CODE REVIEW

**Review by wipfli — APPROVED** [review]
Cool, thanks @lseelenbinder

File: .github/ISSUE_TEMPLATE/new-repo-checklist.md [hunk 1]
```diff
@@ -12,8 +12,9 @@ https://github.com/maplibre/[repo]
 **<...Explain why this repo is good for MapLibre project, its goals, and any other relevant info...>**
 
 ## Acceptance
-- [ ] Any two board members must agree to accept a new repository.
+- [ ] Any two board members must agree to accept a new repository, once the checklist is complete. _Movement to the `maplibre` organization may happen before or after the checklist is complete, but the two "Approved by" board members are required._
```
Comment (HarelM) [inline .github/ISSUE_TEMPLATE/new-repo-checklist.md:15]: Why not complete the checkbox list before moving the repo? Most if not all projects are not linked to their parent fork and one can do the changes before we onboard a project. It shows that there's at least one maintainer that can work on it, doesn't it?

File: .github/ISSUE_TEMPLATE/new-repo-checklist.md [hunk 2]
```diff
@@ -12,8 +12,9 @@ https://github.com/maplibre/[repo]
...
   **Approved by:** <@user1> <@user2>
+- [ ] Denote what tier of project it will be (`Core`, `Supported`, `Hosted`): <...tier...> (See discussion: https://github.com/maplibre/maplibre/discussions/132)
```
Comment (HarelM) [inline .github/ISSUE_TEMPLATE/new-repo-checklist.md:17]: Didn't we agree all project start as hosted? I'm not sure this is needed.
We can explain as part of this template how to get funds for a hosted project or simply link the relevant wiki page.

**Comment by HarelM** [comment]
This was merged too fast. I'm finding some issues with these changes...

**Comment by lseelenbinder** [comment]
@HarelM Yes, agreed. I was hoping for more input, as well. 😄

**Comment by lseelenbinder** [comment]
Let's resolve all the open discussions on this, and then I'll open a new PR with the final changeset for broader discussion. 😄

File: .github/ISSUE_TEMPLATE/new-repo-checklist.md (hunk 1 above)
Comment (lseelenbinder) [reply .github/ISSUE_TEMPLATE/new-repo-checklist.md:15]: That's a good point. My idea was that sometimes it's a bit more work before a project is ready, but would benefit from getting the stamp of "official" before it's fully ready. I.e., the react native fork will need quite a bit of adjustment before it's fully ready, but we know we need it and want it, so it's a bit weird for everyone to do the work on my fork, when we know we will just move it to maplibre/. I see your point, though.

File: .github/ISSUE_TEMPLATE/new-repo-checklist.md (hunk 2 above)
Comment (lseelenbinder) [reply .github/ISSUE_TEMPLATE/new-repo-checklist.md:17]: I didn't think we had agreed to that, but it's a fair enough to default to Hosted and go from there.

**Comment by wipfli** [comment]
I agree that newly onboarded repositories should start in the hosted category.

File: .github/ISSUE_TEMPLATE/new-repo-checklist.md (hunk 1 above)
Comment (wipfli) [reply .github/ISSUE_TEMPLATE/new-repo-checklist.md:15]: I think to create the repository in the MapLibre GitHub org, two Governing Board members should sign off and approve the onboarding checklist.

Then we can adjust things in the repo and collaborate on it in the GitHub org. Closing the onboarding issue should however only happen once the repo is well-onboarded, i.e., many boxes are ticked...

Comment (HarelM) [reply .github/ISSUE_TEMPLATE/new-repo-checklist.md:15]: The problem from my point of view with this approach is that we move a project under maplibre org that does not follow the "rules" we created. Once it's under maplibre org there's no guarantee it will finish the checklist and then we need to ping the maintainer hoping it will be resolved.
I prefer to look at this as a gatekeeping list.
I don't think this checklist is a lot of work in most cases, not compared to regular maintenance.
If we say that this is a pre-requisite it's the maintainer responsibility to do it, without it s.he won't be able to say that it's part of maplibre org.
Bottom line, I prefer to be chased and not to chase :-)

Comment (wipfli) [reply .github/ISSUE_TEMPLATE/new-repo-checklist.md:15]: Good point, Harel.

Comment (HarelM) [reply .github/ISSUE_TEMPLATE/new-repo-checklist.md:15]: BTW, I just took a repo and aligned it to the on boarding checklist. It took me 10 minutes...

Comment (lseelenbinder) [reply .github/ISSUE_TEMPLATE/new-repo-checklist.md:15]: > Bottom line, I prefer to be chased and not to chase :-)

True that. I'm in agreement. :+1:
